HACKING.md

```diff
@@ -94,4 +96,22 @@ To rebuild the containers after a change, add the `--build` flag to the `docker-
 
 ```
 docker-compose up --build
-```
\ No newline at end of file
+```
+
+## Shortening the loop
+
+For some components, it is possible to install distribution packages first and then only to replace binaries which may or may not work for smaller changes.
+
+```
+systemctl stop osbuild-composer.service osbuild-composer.socket osbuild-local-worker.socket
+make build && sudo install -m755 osbuild-composer /usr/libexec/osbuild-composer/
```

Comment:
The install now should be
```
sudo install -m755 bin/osbuild-composer ...
```
Also would probably be useful to also `install ... bin/osbuild-worker` at the same time.